src/buffer.rs

```diff
@@ -80,6 +81,11 @@ impl Cell {
             .add_modifier(self.modifier)
     }
 
+    pub fn set_skip(&mut self, skip: bool) -> &mut Cell {
```

Comment:
It would be good to have some docs on this to help state the purpose of a per cell skip. How about something like: "Sets the cell to be skipped when copying the buffer to the screen. This helps avoid overwriting terminal graphics that would be in that cell (Sixels / iTerm / Kitty graphics)"?

Reply:
Should this be `skip(&mut self, skip: bool)` instead? `skip` can be used as a verb, and `Cell::new(...).skip(true)` _feels_ good to me.

Also, is there a reason we are not using `mut self` here? 

```diff
-     pub fn set_skip(&mut self, skip: bool) -> &mut Cell {
+     pub fn skip(mut self, skip: bool) -> Cell {
```

I see other methods also use `&mut self`. Maybe this is just how the API has always been in this case.

Reply:
The current method is consistent with the other set methods for fg bg etc. That's more important for this right now. Skip is probably not the best name, that might be something unwieldy like `exclude_from_buffer_sync` / `do_not_display` or something similar. But contextually, and with appropriate comments about why it exists, I think skip is a "good enough" name for this feature (unless there's an obvious better short name we are missing).